index.js

```diff
@@ -418,6 +419,15 @@ exports.connect = function(config, intern, callback) {
   type = internals.mod.type;
 
   internals.interfaces.SeederInterface._makeParamArgs = dummy;
+  if( config.hasOwnProperty("ssl") && config.ssl.hasOwnProperty("path") && config.ssl.path ){
```

Comment:
rather use and check directly against `config.ssl` and `config.ssl.path` for existence than using hasOwnProperty.